pmd-java/src/main/java/net/sourceforge/pmd/lang/java/JavaTokenManager.java.bak

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
I think, this file got accidentally added....

Reply:
Yes, sorry, you're right. I've removed it now.